site/templates/application.jet

```diff
@@ -54,6 +54,19 @@
   </head>
   <body>
     <s72-cookie-consent></s72-cookie-consent>
+
+    <section>
```

Comment:
Can we not put the `sr-only` class up here?

Reply:
Yeah.  Weirdly when I tried that before it would still show the <ul> bullet point but it must have just been Chrome flipping out.